nginx_proxy/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.0.1
+
+- Fix the use of subfolders with certificate files
```

Comment:
```suggestion
- Add support for certificate files located in subdirectories
```
Maybe?

Reply:
It's more a fix. Anyway with the cert manager, such issues gone